tests/integration/test_model.py

```diff
@@ -903,7 +903,6 @@ def test_ml_experiment(c, client, training_df):
 
 # TODO - many ML tests fail on clusters without sklearn - can we avoid this?
 @xfail_if_external_scheduler
-@pytest.mark.skip(reason="Waiting on https://github.com/EpistasisLab/tpot/pull/1280")
```

Comment:
cc @sarahyurick since this is your PR; are you aware of any other upstream changes that could've resolved these failures?

Reply:
AFAIK, the only other solution is to restrict the NumPy version to be < 1.24.0